<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManualWindow</class>
 <widget class="QMainWindow" name="ManualWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="auto_button">
    <property name="geometry">
     <rect>
      <x>1700</x>
      <y>875</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>Autonomous</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stitch_button">
    <property name="geometry">
     <rect>
      <x>1700</x>
      <y>550</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>Video Stitching</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stereo_button">
    <property name="geometry">
     <rect>
      <x>1700</x>
      <y>700</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>Stereo Vision</string>
    </property>
   </widget>
   <widget class="QPushButton" name="screenshot_button">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>890</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>Screenshot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backward_button">
    <property name="geometry">
     <rect>
      <x>875</x>
      <y>800</y>
      <width>150</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="forward_button">
    <property name="geometry">
     <rect>
      <x>875</x>
      <y>525</y>
      <width>150</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="left_button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>625</y>
      <width>75</width>
      <height>150</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="right_button">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>625</y>
      <width>75</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="motion">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>550</y>
      <width>150</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>Motion</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="speed">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>665</y>
      <width>150</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="voltage">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>780</y>
      <width>150</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>Voltage</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="current">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>890</y>
      <width>150</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>Current</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="camera">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>500</height>
     </rect>
    </property>
    <property name="text">
     <string>Live Feed</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="motion_pic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>550</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="speed_pic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>665</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="volt_pic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>780</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="current_pic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>890</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>650</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="caption">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>650</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Training'24&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Team 1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="LF">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>525</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="RF">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>525</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="LB">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>800</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="RB">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>800</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Drive">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>525</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Drive</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Park">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>525</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Park</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
